compiler-rt/lib/builtins/CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ set(GENERIC_SOURCES
   divti3.c
   extendsfdf2.c
   extendhfsf2.c
+  extendhfxf2.c
```

Comment:
The xf type is only supported for x86 so it should not be in the generic sources. 

Could you please add me as a reviewer when you submit a new version of this patch.

Reply:
Sure.